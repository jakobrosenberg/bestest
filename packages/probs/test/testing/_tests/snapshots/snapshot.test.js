test('create snapshot', () => {
    expect('foobar').toMatchSnapshot()
})