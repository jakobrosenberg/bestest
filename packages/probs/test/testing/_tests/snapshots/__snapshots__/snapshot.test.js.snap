// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test/testing/_tests/snapshots/snapshot.test.js/create snapshot 1`] = `"foobar"`;
